CommandScreen.c

```diff
@@ -17,7 +17,7 @@ static void CommandScreen_scan(InfoScreen* this) {
    Panel_prune(panel);
 
    const char* p = Process_getCommand(this->process);
-   char* line = xMalloc(COLS + 1);
+   char line[COLS + 1];
    int line_offset = 0, last_spc = -1, len;
    for (; *p != '\0'; p++, line_offset++) {
       line[line_offset] = *p;
```

Comment:
Add extra `assert(line_offset < COLS + 1)` as cautionary measure when converting this to a stack buffer.